TS01-Time-Series_Metadata.Rmd

```diff
@@ -65,37 +60,20 @@ refresher on setting the working directory, please view the <a href="XXX" target
 
 </div>
 
-#Understand Your Data 
```

Comment:
Also this lesson isn't about good coding - it's about time series. So i'd keep the good coding content in the intro materials and focus on the time series here - lead by example. Note small points of good practices throughout. keep it short and distill info where possible. people can only learn so much in one lesson so overload also can be problematic. 2-4 objectives is a good goal. then stick to those throughout.